microsetta_interface/implementation.py

```diff
@@ -1316,6 +1324,27 @@ def get_create_nonhuman_source(*, account_id=None):
                                  account_id=account_id)
 
 
+# Note: ideally this would be represented as a DELETE, not as a POST
+# However, it is used as a form submission action, and HTML forms do not
+# support delete as an action
+def post_request_account_removal(*, account_id, body):
+    # PUT is used to add the account_id to the queue
+    # DELETE is used to remove the account_id from the queue, if it's
+    # still there.
+
+    user_delete_reason = body.get('user_delete_reason')
+
+    url = f'/accounts/{account_id}/removal_queue' \
+          f'?user_delete_reason={user_delete_reason}'
+
+    has_error, put_output, _ = ApiRequest.put(url)
+
+    if has_error:
+        return put_output
+
+    return _render_with_defaults('request_account_deletion_confirm.jinja2')
```

Comment:
@ayobi I'm running into an odd issue with this on the staging server. Private API receives and successfully logs the removal request, but when the template contents are returned, they're enclosed in double quotes and the browser isn't rendering it correctly (screenshot below). Does this match what's happening in your local environment? I'm trying to determine if this is a code issue with the template/yaml file or what's going on here.

![request_removal_raw](https://github.com/biocore/microsetta-interface/assets/83246693/5fe7b7cf-7ac1-4ed4-84d8-cd93f6b86ef3)